backend/src/main/kotlin/blue/mild/covid/vaxx/service/dummy/DummyIsinService.kt

```diff
@@ -16,7 +16,7 @@ class DummyIsinService : IsinServiceInterface {
         personalNumber: String
     ): IsinGetPatientByParametersResultDto {
         logger.warn { "NOT GETTING patient ${firstName}/${lastName}/${personalNumber} from ISIN. This should not be in the production." }
-        return IsinGetPatientByParametersResultDto("PacientNebylNalezen", null, null)
+        return IsinGetPatientByParametersResultDto("Chyba", null, null)
```

Comment:
Skoro bych rekl, ze `PouzitiDummyISIN` je lepsi, ne?

Reply:
není to hodnota toho enumu co vrací ISIN, ale vlastně může být